src/main/java/com/google/cloud/tools/app/impl/config/DefaultVersionsSelectionConfiguration.java

```diff
@@ -26,10 +26,20 @@
  */
 public class DefaultVersionsSelectionConfiguration implements VersionsSelectionConfiguration {
 
+  private String project;
   private Collection<String> versions;
   private String service;
 
   @Override
+  public String getProject() {
```

Comment:
If we do this, then we can no longer code the CloudSdk action classes against configuration interfaces